Client/Clique/Common/Extensions/UIViewController+Extension.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  UIViewController+Extension.swift
+//  Clique
+//
+//  Created by Infinum on 09.12.2022..
+//
+
+import UIKit
+
+extension UIViewController {
+
+    func sendOkAlert(
+        with title: String = "",
+        message: String
+    ) {
+        let alertController = UIAlertController(title: title, message: message ,
+                                                preferredStyle: .alert)
+        let okAction = UIAlertAction(title: Constants.Alerts.defaultOKActionTitle,
+                                     style: .cancel)
+        alertController.addAction(okAction)
+        present(alertController, animated: true)
+    }
+
+    func sendAlert(
+        with title: String = "",
+        message: String,
+        action: UIAlertAction
+    ) {
+        let alertController = UIAlertController(title: title, message: message,
+                                                preferredStyle: .alert)
+        alertController.addAction(action)
+        present(alertController, animated: true)
+    }
+    
+    func sendOKCancelAlert(
+        with title: String = "",
+        message: String,
+        actions: [UIAlertAction]
+    ) {
+        let alertController = UIAlertController(title: title, message: message,
+                                                preferredStyle: .alert)
+        actions.forEach { alertController.addAction($0) }
+        present(alertController, animated: true)
+    }
+}
```

Comment:
You do not need a method which accepts an action and then other method which accepts multiple actions. Remove the first one, also take notes in formatting. If you have init methods with multiple parameters please put each parameter in it's own line!
<img width="625" alt="Screenshot 2022-12-14 at 09 11 02" src="https://user-images.githubusercontent.com/67597006/207541531-528cc1d2-b8f9-464e-ae4b-4dfbdf2f4122.png">